integration-tests/msa/createMsa.test.ts

```diff
@@ -1,69 +1,70 @@
 import "@frequency-chain/api-augment";
 import assert from "assert";
 import { ApiRx } from "@polkadot/api";
-import { connect, createKeys } from "../scaffolding/apiConnection"
-import { groupEventsByKey, signPayloadSr25519 } from "../scaffolding/helpers";
+import { connect } from "../scaffolding/apiConnection"
+import { createAccount, createAndFundAccount, DevAccounts, INITIAL_FUNDING, showTotalCost, signPayloadSr25519 } from "../scaffolding/helpers";
 import { KeyringPair } from "@polkadot/keyring/types";
-import { addPublicKeyToMsa, createMsa, createSchema, deletePublicKey } from "../scaffolding/extrinsicHelpers";
+import { addPublicKeyToMsa, createMsa, createSchema, deletePublicKey, signAndSend } from "../scaffolding/extrinsicHelpers";
 import { AVRO_GRAPH_CHANGE } from "../schemas/fixtures/avroGraphChangeSchemaType";
-import { filter, firstValueFrom } from "rxjs";
 
-describe("Create Accounts", () => {
+describe("Create Accounts", function () {
+    this.timeout(150000);
```

Comment:
Is this 150 seconds?  Is that too long?  Is there a default timeout value constant for all tests (that can be overridden if needed)?

Reply:
Oops... that should be `15000`, not `150000`. Fixing.